src/org/zaproxy/zap/extension/pscanrulesAlpha/ZapAddOn.xml

```diff
@@ -7,6 +7,7 @@
 	<url/>
 	<changes>
 	<![CDATA[
+		PiiScanner add word boundry checks to reduce false positives.<br>
```

Comment:
@kingthorin the typo was here as well (just noticed in the migration), mind opening a PR?

Reply:
On it.